<launch>

  <arg name="do_sim" default="true" description="Simulate cmd_vel->odom update"/>

  <!-- launch algorithm node -->
  <node pkg="brne_py" exec="brne_nav">
  </node>

  <!-- simulate cmd_vel to odom updates -->
  <node pkg="crowd_nav" exec="sim_robot" if="$(eval '\'$(var do_sim)\' == \'true\'')">
  </node>

  <!-- simulate a pedestrian at 1,0 -->
  <node pkg="pedestrian_tracking" exec="simulate_pedestrians" if="$(eval '\'$(var do_sim)\' == \'true\'')">
  </node>

   <!-- publish pedestrian markers -->
  <node pkg="pedestrian_tracking" exec="show_pedestrians">
  </node>

  <node pkg="rviz2" exec="rviz2" args="-d $(find-pkg-share crowd_nav)/config/odom_test.rviz"/>

</launch>
